Sources/NIOCore/EventLoop+SerialExecutor.swift

```diff
@@ -23,11 +23,12 @@ public protocol NIOSerialEventLoopExecutor: EventLoop, SerialExecutor { }
 
 @available(macOS 14.0, iOS 17.0, watchOS 10.0, tvOS 17.0, *)
 extension NIOSerialEventLoopExecutor {
-    @inlinable
-    public func enqueue(_ job: consuming ExecutorJob) {
```

Comment:
Why did this get removed?

Reply:
I thought I had to drop down to `UnownedJob`s but I think I made it work now.